docs/source/getting_started/faq.rst

```diff
@@ -0,0 +1,25 @@
+Frequently Asked Questions (FAQ)
+================================
+
+Below are some frequently asked questions regarding the beta release
+series. Click on a question to be directed to relevant information in
+our documentation or our GitHub repo.
+
+General
+-------
+
+-  :ref:`What is JupyterLab? <overview>`
+-  :ref:`How stable is JupyterLab? <stability>`
```

Comment:
Meta question- Do we need to keep this question in the FAQ now that we're putting this all over other parts of the docs?